hasura/migrations/default/1694549051704_alter_table_public_users_add_column_profile_id/up.sql

```diff
@@ -0,0 +1,7 @@
+alter table "public"."users"
+add column "profile_id" bigint;
+UPDATE "public"."users"
+SET profile_id = profiles.id
+FROM "public"."profiles"
+WHERE "public"."users".address = profiles.address;
```

Comment:
make sure this is case insensitive since addresses are sometimes cased wierdly in our codebase @sshmm

Reply:
looks like this got wrapped with lower()